java/tools/src/test/org/apache/orc/tools/TestJsonFileDump.java

```diff
@@ -159,7 +159,7 @@ public void testDoubleNaNAndInfinite() throws Exception {
     FileOutputStream myOut = new FileOutputStream(workDir + File.separator + outputFilename);
 
     // replace stdout and run command
-    System.setOut(new PrintStream(myOut, true, StandardCharsets.UTF_8));
+    System.setOut(new PrintStream(myOut, true, StandardCharsets.UTF_8.toString()));
```

Comment:
Looks good, does it make sense to have the same fix here and in main branch? The above API is available in both JDK 8 and 17, and the Charset one is only available in JDK17. Just a suggestion, else fix looks good

Reply:
No need for main/branch-2.0 because the minimum is Java 17